app/views/application/_nav.html.slim

```diff
@@ -27,6 +27,7 @@ nav.navbar.navbar-default.navbar-static-top.top-nav-collapse
                       = t ".security"
                   - if show_faq_link?
                     li= link_to t(".faqs"), faqs_path
+                  li= link_to t(".help"), "https://support.brave.com/hc/en-us/"
```

Comment:
In the future we should sending users to the locale that they're currently on

Reply:
Yup good idea, as of right now HC is only en-us 🇺🇸